src/applications/gi/sass/gi.scss

```diff
@@ -71,4 +71,11 @@
   select.hide-arrows {
     background-image: none
   }
+
+  .search-result {
+    i.fa {
+      background: initial;
+      color: initial;
+   }
+  }
```

Comment:
I'm curious where the rule is that changed the colors and necessitates this new rule.

Reply:
There are some new merger related styles for icons in general and for specific types of icons.
![image](https://user-images.githubusercontent.com/16051172/46030173-4b979a00-c0aa-11e8-8385-08862befdd4d.png)